python/GafferScene/ScriptProcedural.py

```diff
@@ -167,4 +169,27 @@ def __allRendered():
 		Gaffer.ValuePlug.setCacheMemoryLimit( 0 )
 		Gaffer.ValuePlug.setCacheMemoryLimit( memoryLimit )
 
+	__errorConnections = {}
+	@classmethod
+	def __ensureErrorConnection( cls, node ) :
+
+		if node in cls.__errorConnections :
+			return
+
+		cls.__errorConnections[node] = node.errorSignal().connect( cls.__error )
+
+	@staticmethod
+	def __error( plug, source, error ) :
+
+		errorContext = "Plug \"%s\"" % source.relativeName( source.ancestor( Gaffer.ScriptNode ) )
+		if "scene:path" in Gaffer.Context.current() :
+			path = "/" + "/".join( [ str( x ) for x in Gaffer.Context.current()["scene:path"] ] )
```

Comment:
Mostly because that method wasn't on my radar, but yes, it does seem odd to rely on that in this context. I've fixed up the bindings for `ScenePlug::pathToString()` and used that instead.